.env.demo

```
@@ -1,140 +1,136 @@
-POOL_DATABASE_URL="postgresql://postgres:postgres@your-ip:5432/credebl"
-DATABASE_URL="postgresql://postgres:postgres@your-ip:5432/credebl"
+POOL_DATABASE_URL="postgresql://postgres:postgres@your-ip:5432/credebl" # Provide pooler supabase postgres URL
+DATABASE_URL="postgresql://postgres:postgres@your-ip:5432/credebl" # Provide supabase postgres URL and Use the correct user/pwd, IP Address
 
 # Used for Bulk issuance of credential
 # Optional (Can be skipped if Bulk issuance is not used)
-AWS_ACCESS_KEY=
-AWS_SECRET_KEY=
-AWS_REGION=
-AWS_BUCKET=
+AWS_ACCESS_KEY= # Please provide your access key
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)